LOCALDEV.md

```diff
@@ -76,10 +78,10 @@ bundle install
 
 *NOTE:* You may encounter issues installing gems. If you do, investigate the errors. You may require additional libraries that can be install with homebrew.
 
-Next, the database needs to be created. If Rails is installed correctly, you this command should create the `ceqr_rails` database and load the schema.
+Next, the database needs to be created. If Rails is installed correctly, you this command should create the `ceqr_rails` database, load the schema, and seed the db with some initial data.
```

Comment:
nit: clarify `... correctly, you this command...`

Reply:
Thanks for reviewing and finding these typos!  Fixed them both here:  https://github.com/NYCPlanning/ceqr-app/pull/501/commits/9016484811bddb3583d422eec08e2cb1f9e6448f.